src/main.rs

```diff
@@ -209,16 +221,16 @@ mod tests {
         )
         .unwrap();
 
-        let package_manifest = read_package_manifest(&package_json_path).unwrap();
+        let received = read_package_manifest(&package_json_path).unwrap();
 
-        let received = serde_json::to_string_pretty(&package_manifest).unwrap();
         let expected = serde_json::json!({
             "scripts": {
                 "test": "echo hello world"
             }
         })
         .pipe_ref(serde_json::to_string_pretty)
         .unwrap();
+        let expected: NodeManifest = serde_json::from_str(expected.as_str()).unwrap();
```

Comment:
You can convert `Value` directly to `NodeManifest` without converting it to string.

```rust
        let expected = serde_json::json!({
            "scripts": {
                "test": "echo hello world"
            }
        });
        let expected: NodeManifest = serde_json::from_value(expected).unwrap();
```

You should also remove the `serde_json::` qualifier (adding `use serde_json::json` to the top):

```rust
        let expected = json!({
            "scripts": {
                "test": "echo hello world"
            }
        });
        let expected: NodeManifest = serde_json::from_value(expected).unwrap();
```

Reply:
In case you are wondering "why not also remove `serde_json::` qualifier from `to_string_pretty`?", my answer is `to_string_pretty` is an ambiguous name which is also used by other serde crates.

Reply:
fixed that. anything else needed to change?